programs/clearing_house/src/math/position.rs

```diff
@@ -84,10 +84,33 @@ pub fn calculate_base_asset_value(
         .checked_sub(amm.user_lp_shares)
         .ok_or_else(math_error!())?;
 
-    let base_asset_reserve_proportion =
```

Comment:
can we change the name of this function if it's only used for the amm now? kinda misleading to have such a general name like `calculate_base_asset_value` since its only used for amm math now and less general purpose cuz of lp logic

Reply:
can you also get rid of the `use_spread` in this function, nobody ever calls it with it set to true